LPS-136379 TEST

stsquared99 opened the pull request.

ctang3 committed:
LPS-131511 Create CTMappingTableInfo to store CTCollection mapping table changes

ctang3 committed:
LPS-131511 Extract interface

ctang3 committed:
LPS-131511 buildService

Preston-Crary committed:
LPS-131511 Avoid creating extra CTMappingTableInfoImpl objects. Simplify exception handling, use runtime exception since callers cannot handle SQLExceptions and they should not happen. Use column index instead of name since it's clear enough here.

Preston-Crary committed:
LPS-131511 Build Service.

Preston-Crary committed:
LPS-131511 Add packageinfo.

marcoscv committed:
LPS-140602 Add site initializer module

marcoscv committed:
LPS-140602 Fix names

marcoscv committed:
LPS-140602 New privacy policy

marcoscv committed:
LPS-140602 Add widget ADTs

marcoscv committed:
LPS-140602 Pages update

marcoscv committed:
LPS-140602 Add page JS

Commit:
LPS-140602 Fix className

Commit:
LPS-140602 Use JS from js.js file

Commit:
LPS-140602 SF

Commit:
LPS-140602 Fix language key

marcoscv committed:
LPS-140602 Fix js.js SF

atjuan committed:
LPS-140602 Create master pages

atjuan committed:
LPS-140602 Create pages

atjuan committed:
LPS-140602 Remove privacy and documentation pages from footer navigation

kyle-miho committed:
LRQA-71581 Explicitly create null convertable string entity

kyle-miho committed:
LRQA-71581 buildService

Commit:
LPS-140602 Load Commerce Data

Commit:
LPS-140602 Add Vocabulary and Categories

Commit:
LPS-140602 Add Footer Nav Menu

Commit:
LPS-140602 Fix views

Commit:
LPS-140602 Fix ADTs

Commit:
LPS-140602 Fix URLs

Commit:
LPS-140602 Remove redundant headers

jkappler committed:
LPS-143422 Hide private pages options in pages tree if there are no private pages in the site

jkappler committed:
LPS-143422 Rely for now on ServiceContext to change the title of the tab if there are no private pages

jkappler committed:
LPS-143422 Hide private pages in navigation menus

jkappler committed:
LPS-143422 Don't show private pages in selector when there are no in the site

jkappler committed:
LPS-140253 Apply usage

christian-chiappa committed:
COMMERCE-7742 update definition service to handle virtual product type

christian-chiappa committed:
COMMERCE-7742 update dependencies in build.gradle

christian-chiappa committed:
COMMERCE-7742 update integration test and dependencies

christian-chiappa committed:
COMMERCE-7742 shippable disabled for virtual product

jkappler committed:
LPS-140253 New Helper to check if we want to show private pages for a given site

jkappler committed:
LPS-140253 New Util class with a service proxy so that we can check if the privates are visible for a given site

christian-chiappa committed:
COMMERCE-7742 added dependency on jps

christian-chiappa committed:
COMMERCE-7742 added dependency on build.gradle

jkappler committed:
LPS-140253 Adds feature flag for disabling private pages

jkappler committed:
LPS-140253 Only hide private pages if the feature flag is disabled

christian-chiappa committed:
COMMERCE-7742 Semantic versioning (gw baseline)

MarinhoFeliphe committed:
LPS-137019 Create AddTaskRequest model in metrics

MarinhoFeliphe committed:
LPS-137019 Add new indexer task method

MarinhoFeliphe committed:
LPS-137019 Apply new indexer task method to use

izaera committed:
LPS-142295 Create sample Vanilla Counter remote app on DB creation

We create a sample row pointing to our Vanilla Counter app per company,
but only when the table is empty (ie: on fresh installations).

Note that this will recreate the sample if it is deleted, no other
remote app is created and portal is restarted. PM thinks this is a fair
behaviour but, if we see it creates any problem, we may change it in
the future.

izaera committed:
LPS-142295 Add a method to count RemoteAppEntry instances by companyId

MarinhoFeliphe committed:
LPS-137019 Create CompleteTaskRequest

MarinhoFeliphe committed:
LPS-137019 Create new complete task method

jkappler committed:
LPS-140253 Baseline

MarinhoFeliphe committed:
LPS-137019 Create UpdateTaskRequest

MarinhoFeliphe committed:
LPS-137019 Add new update task method

magjed4289 committed:
LPS-142295 test-fix RemoteApps#CanBeDeleted

christian-chiappa committed:
COMMERCE-7742 reorder setShippable

christian-chiappa committed:
COMMERCE-7742 virtual clause

MarinhoFeliphe committed:
LPS-137019 Update integration tests

MarinhoFeliphe committed:
LPS-137019 Apply new complete task method to use

MarinhoFeliphe committed:
LPS-137019 Apply new update task method to use

MarinhoFeliphe committed:
LPS-137019 Create DeleteTaskRequest

MarinhoFeliphe committed:
LPS-137019 Create new delete task method

MarinhoFeliphe committed:
LPS-137019 Apply new delete task method to use

arthurchan35 committed:
LPS-138196 Revert "LPS-138196 Do not clean up before all nodes are done"

This reverts commit f268c58826514dede344be30cb68a5eda79940b2.

arthurchan35 committed:
LPS-138196 Properly handle deletion as oidc session is /user/configuration

arthurchan35 committed:
LPS-138196 Regen

arthurchan35 committed:
LPS-138196 Allow to find entry by userId and ProviderName

arthurchan35 committed:
LPS-138196 Reuse session entry

MarinhoFeliphe committed:
LPS-137019 Semver

izaera committed:
LPS-142295 Add ResourceBundle for RemoteAppEntry portlets

This is so that the portlet title is correctly localized

Commit:
LPS-134172 Add {relationshipName}Id fields to GraphQL

Commit:
LPS-134172 Add {relationshipName}Id field to the ObjectEntryEntityModel to be able to filter by relationship id

MarinhoFeliphe committed:
LPS-142443 Add label field

MarinhoFeliphe committed:
LPS-142443 Build Service

Commit:
LPS-136379 Versioning

Commit:
LPS-136379 Generate mapping info in review changes

andrea-ale-sbarra committed:
COMMERCE-7742 commerce-product-service sorted instructions

MarinhoFeliphe committed:
LPS-142443 Apply label element on the nodes

Commit:
LPS-136379 Move toolbar into content area

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

kyle-miho committed:
LRQA-71581 Implement persistence methods needed to expose bug

kyle-miho committed:
LRQA-71581 Add integration tests to cover issues with null convertible strings

kyle-miho committed:
LRQA-71581 buildService

MarinhoFeliphe committed:
LPS-142443 Add label map in Node class

MarinhoFeliphe committed:
LPS-142443 Parse label

MarinhoFeliphe committed:
LPS-142443 Persist node label

austinchiang committed:
LPS-136390 Rename macro

austinchiang committed:
LPS-136390 Improve test design to account for Google server errors

marianoalvarosaiz committed:
LPS-138464 Continue applying SearchOrderByUtil pattern.

izaera committed:
LPS-142295 gradle buildService

marianoalvarosaiz committed:
LPS-138464 SF

MarinhoFeliphe committed:
LPS-137019 Remove unused methods

MarinhoFeliphe committed:
LPS-142443 Add label field to WorkflowNode

izaera committed:
LPS-142295 Fix race condition and add description

We need to add a reference to the WorkflowHandler because otherwise the remote
service fails (sometimes) because it cannot be found. This doesn't happen with
regular services but given that the listener is invoked very soon there seems
to be a race condition in OSGi initialization.

MarinhoFeliphe committed:
LPS-142443 Set label to workflow engine impl

MarinhoFeliphe committed:
LPS-142443 Apply label from WorkflowNode to Node DTO

Commit:
COMMERCE-7923 build service

Commit:
COMMERCE-7923 baseline

Commit:
LPS-136379 Add data/relationships tabs

Commit:
LPS-142243 Add Fetch delete request handler

Commit:
LPS-142243 Add FetchMock class to mock fetch request

Commit:
LPS-142243 Change fetch mock on tests

Commit:
LPS-136379 Add navigation to url history

Commit:
LPS-136833 Add test to check that a guest cannot add a document in a folder without permissions

peerkar committed:
LPS-143202 search-experiences-service: Add methods for deleting company items

peerkar committed:
LPS-143202 search-experiences-api: gw buildService

peerkar committed:
LPS-143202 search-experiences-service: Check on activate that default elements exist in each company. Delete data on un-unregister

Commit:
LPS-136379 Display relationships table

brianchandotcom committed:
LPS-143202 do we really need Modified?

brianchandotcom committed:
LPS-143202 Your current logic will read all the default files on activate AND every time a new company is added. Let's just read those files once.

brianchandotcom committed:
LPS-143202 we have call the forEach method (which is why SF was complaining) because this is a security issue. Calling forEach sets certain thread variables to ensure that in a SaaS env, company A does not touch company B. In this case, it's not really a problem at all. SF is overly aggressive, but it's still a best practice.

brianchandotcom committed:
LPS-143202 SF

brianchandotcom committed:
LPS-143202 This synchronization is not useful. In a clustered environment... we'd have trouble because different nodes are running on different JVMs. We need to revisit this whole pattern.

austinchiang committed:
LPS-136390 Remove list from macro

shuyangzhou committed:
LRQA-71581 Not needed

shuyangzhou committed:
LRQA-71581 More coverage

shuyangzhou committed:
LRQA-71581 Be strict.

Commit:
COMMERCE-7970 build lang

Commit:
COMMERCE-7923 fix integration tests

Commit:
LPS-136834 Add test to check that a guest cannot delete a file without permissions

Commit:
LPS-142295 Add status as a method parameter for addCustomElementRemoteAppEntry to bypass the Workflow framework when need it

Commit:
LPS-142295 Create the sample Vanilla Counter remote app and Bundle Initializer custom elements with approved status to avoid calling the Workflow framework

Commit:
LPS-136845 Add test to check that a site member cannot delete a file without permissions

marcelabc committed:
LPS-142893 Change to compileOnly

marcelabc committed:
LPS-142893 Fix the Clob data type for databases compatibility

marcelabc committed:
LPS-142893 Add condition when Clob field value is null

MarinhoFeliphe committed:
LPS-142443 gw buildUpgradeTable

MarinhoFeliphe committed:
LPS-142443 Create upgrade to create label column

MarinhoFeliphe committed:
LPS-142443 Export labels

MarinhoFeliphe committed:
LPS-142443 Semver

brookedalton committed:
LRQA-69578 Add testcase to assert synonyms aren't deleted after reindexing

Commit:
LPS-136841 Add test to check that a site member cannot add a document in a folder without permissions

hhuijser committed:
LPS-143349 Add xmlDefinitions information to JSON

hhuijser committed:
LPS-143349 Add check to detect removed XML definitions when upgrading

shuyangzhou committed:
LPS-143202 This is not needed, PortalInstanceLifecycleListener is stateful, it won't miss any pre-exist companies.

shuyangzhou committed:
LPS-143202 Stream styling is evil, you were calling "_sxpElementLocalService.getSXPElements()" 39 times per company without knowing it.

hhuijser committed:
LPS-143349 Generated

Jeremy-Ch committed:
LPS-136833 Reorder test steps

Jeremy-Ch committed:
LPS-136833 Add macro for viewing no add permission

Jeremy-Ch committed:
LPS-136833 Use a suitable assertion

Jeremy-Ch committed:
LPS-136834 Reorder test steps

Jeremy-Ch committed:
LPS-136834 Add macro for viewing no specific items in the management bar

Jeremy-Ch committed:
LPS-136834 Use suitable macros to assert

Jeremy-Ch committed:
LPS-136841 Use suitable macros

Jeremy-Ch committed:
LPS-136845 Add macro for viewing specific disabled items in the management bar

Jeremy-Ch committed:
LPS-136845 Update test to use suitable macros

Jeremy-Ch committed:
LPS-136841 Update test for viewing more menu item

Commit:
LPS-143334 Just shortcut to Object.toString instead of String.valueOf

Commit:
LPS-142295 Build Service

Commit:
LPS-134172 Change relationship name to refer to the one side of the relationship

Commit:
LPS-143334 GetterUtil.getString returns null for a non-String object

claraizquierdo committed:
LPS-143333 Unify p_s_e_id and segmentsExperienceId to fix the edition of variants in ab testing

ealonso committed:
LPS-140253 Moves code to GroupImpl, copy pattern from OrganizationImpl#UserFileUploadsSettings

ealonso committed:
LPS-140253 Rename to LayoutVisibilityManager

ealonso committed:
LPS-140253 baseline

hhuijser committed:
LPS-143349 Add missing import

hhuijser committed:
LPS-143349 Move method getPackageName to JavaSourceUtil

hhuijser committed:
LPS-143349 Add logic to retrieve fullyQualifiedName for type name

hhuijser committed:
LPS-143349 Use constants

hhuijser committed:
LPS-143349 Find imports that no longer exist in upgrade version

hhuijser committed:
LPS-143349 Find method calls to methods that no longer exist in upgrade version

hhuijser committed:
LPS-143349 Generated

Commit:
LPS-140602 Place fragment files in a folder named as the collection key

Commit:
LPS-140602 Update navigation menus

Commit:
LPS-140602 Add vocabulary name

thaiscabrals committed:
LPS-142183 Add Macros, Paths and Tests for Raylife Application Navigation

thaiscabrals committed:
LPS-142183 SF

michaelhashimoto committed:
LRCI-2656 Single thread writing the properties files for workspace git repositories

Commit:
LPS-140602 Move and rename

marcelabc committed:
LPS-142893 Add tests for the Clob type

brianchandotcom committed:
COMMERCE-7923 wordsmith

brianchandotcom committed:
COMMERCE-7923 regen

brianchandotcom committed:
COMMERCE-7742 Simplify

brianchandotcom committed:
COMMERCE-7742 Simplify

xbrianlee committed:
LRQA-69578 Minor formatting

brianchandotcom committed:
COMMERCE-7742 rename and move it closer to where it's used

brianchandotcom committed:
COMMERCE-7742 Consistency

brianchandotcom committed:
LPS-142295 Rename, service methods look like this

brianchandotcom committed:
LPS-142295 sort

brianchandotcom committed:
LPS-142295 regen

brianchandotcom committed:
RELEASE-4166 7.3.10 SP3

thaiscabrals committed:
LPS-142183 Fix Errors

brianchandotcom committed:
LPS-142295 apply

brianchandotcom committed:
LPS-142295 don't create an unnecessary intermediate list

anabuchmann committed:
LRQA-70879 - Add WorkflowPageRevision tests

brianchandotcom committed:
LPS-142295 simplify

arboliveira committed:
LPS-143406 search-experiences-rest-impl: search, like every endpoint, must sanitize a blueprint hydrated from payload before use

https://issues.liferay.com/browse/LPS-143406

brianchandotcom committed:
LPS-142295 Wordsmith

brianchandotcom committed:
LPS-142893 you only need bumpers in the case where the random values must be unique. In this case, the chances of a 1500 random string being duplicate is close to 0. And, even if it was duplicate, it wouldn't matter because the test here doesn't care.

brianchandotcom committed:
LPS-142893 not needed

brianchandotcom committed:
LPS-142893 this test is not really needed. testUpdateCustomObjectField does not test updating every column type from A to B. We just tested a few and that's good enough. Your change in line 276 is useful though.

xbrianlee committed:
LRQA-72062 Add missing @Priority values

Commit:
LPS-141599 Add CannotAddObjectWithoutAddPermission test

pei-jung committed:
COMMERCE-7489 Add breaking change for LPS-125991

brianchandotcom committed:
LPS-138464 not needed

brianchandotcom committed:
LPS-138464 as used

brianchandotcom committed:
LPS-143202 don't delete sxp elements / blueprints every time you shutdown a Tomcat

PauloVAS committed:
LPS-141629 Add CompletedEntryDisplaysApprovedStatus Test

PauloVAS committed:
LPS-141629 SF

brianchandotcom committed:
LPS-143202 Only list read only ones

brianchandotcom committed:
LPS-143202 regen

brianchandotcom committed:
LPS-143202 only used once

brianchandotcom committed:
LPS-143202 auto SF

brianchandotcom committed:
LPS-143202 TODO move to an upgrade process / model listener

pyoo47 committed:
LRCI-2667 app.server.type is the important indicator

brianchandotcom committed:
LPS-134172 need it for JSPs to compile

brianchandotcom committed:
LPS-143349 prep next

brianchandotcom committed:
LPS-143349 prep next

brianchandotcom committed:
LRCI-2656 prep next

brianchandotcom committed:
LRQA-71581 rename

brianchandotcom committed:
LRQA-71581 regen

ealonso committed:
LPS-143422 Auto

ealonso committed:
LPS-143422 Moves method to layout, so we can reuse it

ealonso committed:
LPS-143422 Uses new method and removes duplicated code

liferay-translation committed:
LPS-77699 Update Translations

lfbesada committed:
LPS-142917 Avoiding use infoForm.getName() as className.

Currently, all InfoForm implementations use the className as their name, but although it is a convention, it is not a rule. Since we don't know if there are any custom implementations that didn't follow this convention, it is more prudent to change it so that we get the className using GenericUtil.getGenericClass.

ealonso committed:
LPS-143422 baseline

ealonso committed:
LPS-143422 Uses group method to get the correct layout root node name

ealonso committed:
LPS-143422 Update getLayoutRootNodeName to take into account if the private layouts are enabled or not

ealonso committed:
LPS-143422 Removes no necessary language key

ealonso committed:
LPS-143422 buildLang

ealonso committed:
LPS-143422 Rely on layout root node name and reuse go-to-x key

ealonso committed:
LPS-143422 Extract into a new method

ealonso committed:
LPS-143422 Update item selector for navigation menu to take into account if the private layouts are enabled or not

liferay-translation committed:
LPS-77699 Update Translations

ealonso committed:
LPS-143422 Moves code to display context

ealonso committed:
LPS-143422 Update navigation menu to take into account if the private layouts are enabled or not

ealonso committed:
LPS-143422 Inline

ealonso committed:
LPS-143422 Also update the sub title of layout navigation menu item

ealonso committed:
LPS-143422 No needed since we update getLayoutRootNodeName method

ealonso committed:
LPS-143422 Update the theme information for style books

ealonso committed:
LPS-143422 Rename to isPrivateLayoutsEnabled

ealonso committed:
LPS-143422 SF

ealonso committed:
LPS-143422 Update content page editor to reflect is the private pages are enabled or not

liuxingye committed:
LPS-136390 Go to translation admin by URL

brianchandotcom committed:
LPS-143334 Rename

brianchandotcom committed:
LPS-143422 Rename

brianchandotcom committed:
LPS-138196 Our usual order

brianchandotcom committed:
LPS-138196 regen

Jeremy-Ch committed:
LPS-143434 Use the correct site name

Jeremy-Ch committed:
LPS-143434 Use a suitable way to save

Jeremy-Ch committed:
LPS-143436 Remove the unnecessary step

Jeremy-Ch committed:
LPS-143437 Rename tab name

Jeremy-Ch committed:
LPS-143438 Update macro to click the select button

Jeremy-Ch committed:
LPS-143440 Remove the unnecessary property

liferay-translation committed:
LPS-77699 Update Translations

Jeremy-Ch committed:
LPS-143441 Add macro for go to edit depot permission

Jeremy-Ch committed:
LPS-143441 Update test for searching depot without view permission

Commit:
LPS-136379 Get data source when missing

Commit:
LPS-136379 Fix cursor
